message ssl_decision_packet {
	optional string comment = 1;
}
